style.css

```diff
@@ -0,0 +1,317 @@
+  display: block;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: "";
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+html {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  line-height: 1;
+  font-family:
+    "Lucida Grande",
+    "Lucida Sans Unicode",
+    "Lucida Sans",
+    "Lucida Sans Regular",
+    Geneva,
+    Verdana,
+    sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.container {
+  background: url(https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg);
+  background-repeat: no-repeat;
+  background-position-y: 100%;
+  height: 540px;
+  overflow: hidden;
+  position: relative;
+  text-align: center;
+  color: white;
+  margin-bottom: 0;
+}
+
+.navbar {
+  background:
+    linear-gradient(
+      to bottom,
+      rgb(134, 134, 134) 5%,
+      rgb(128, 128, 128) 50%,
+      rgb(107, 107, 107) 51%,
+      rgb(99, 99, 99) 8%,
+      rgb(121, 121, 121) 94%
+    );
+  width: 980px;
+  height: 35px;
+  position: absolute;
+  margin-left: auto;
+  margin-right: auto;
+  left: 0;
+  right: 0;
+  box-shadow: rgba(0, 0, 0, 0.3) 0 1px 2px;
+  border-radius: 4px;
+  top: 80px;
+  font-size: 16px;
+  border: 1px solid rgb(109, 109, 109);
+}
+
+.navbar-menu {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  justify-items: center;
+  top: 20px;
+}
+
+.nav-item {
+  z-index: 2;
+  width: 100%;
+  height: 100%;
+  position: relative;
+  border-right: 1px solid rgba(75, 75, 75, 0.5);
+  cursor: pointer;
+}
+
+.nav-item span {
+  position: relative;
+  top: 8px;
+}
+
+.nav-item svg {
+  position: relative;
+  top: 5px;
+}
+
+.nav-item:nth-child(9) {
+  position: relative;
+}
+
+.nav-item:not(:nth-child(9)):hover {
+  background: radial-gradient(rgb(53, 52, 52), rgb(83, 83, 83));
+}
+
+.nav-item:nth-child(9) input {
+  border-radius: 20px;
+  width: 100px;
+  margin: 5px 0 0 0;
+  background: transparent;
+  transition: all 0.3s ease-in-out;
+}
+
+.nav-item:nth-child(9) svg {
+  float: left;
+  top: -17px;
+  left: 10px;
+}
+
+.main-content {
+  top: 150px;
+  position: relative;
+  left: 0;
+  right: 0;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.title {
+  font-size: 4em;
+  font-weight: 100;
+  font-family: "Lato", sans-serif;
+  padding-bottom: 1.6rem;
+}
+
+.subtitle {
+  font-weight: 100;
+  font-size: 2rem;
+  font-family: "Lato", sans-serif;
+  width: 63%;
+  padding-bottom: 1.6rem;
+  line-height: 2.6rem;
+}
+
+.sub-title-2 {
+  height: 32px;
+  overflow: hidden;
+  position: relative;
+}
+
+.sub-title-2 img:hover {
+  position: relative;
+  top: -32px;
+}
+
+.asidebar ul {
+  display: flex;
```

Comment:
Let's add `width` property to the `li.aside-item` and `li.aside-item img` elements here.

The images are taking more space than they should.

![apple-foot-2](https://user-images.githubusercontent.com/36057474/65810826-9a897b00-e1a7-11e9-8cbd-469786c3dd48.gif)